### PR TITLE
Add ferengi_2

### PR DESCRIPTION
**Pull request by astopy**

### CODE REVIEW

File: app/lib/config.coffee [hunk 1]
```diff
@@ -147,6 +147,9 @@ Config =
       sdss_lost_set:
         id: '56f2b5ed5925d9004200c775'
         workflowId: '50251c3b516bcb6ecb000002'
+    ferengi_2:
+        id: '58417dcb9afc3a007d000001'
+        workflowId: '5249cbc33ae74070ed000001'
```
Comment (CKrawczyk) [inline app/lib/config.coffee]: Should this be indented to match the other workflows?

Comment (astopy) [inline app/lib/config.coffee]: Yes! Nice catch. I've fixed that now.

**Comment by murraycu** [comment]
Is a new survey on the way?

**Comment by CKrawczyk** [comment]
There is @murraycu, I was going to tag you before accepting the PR.

**Comment by CKrawczyk** [comment]
@astopy Could you also add blank entries in the `developmentLocal` and `test` objects?  This also needs to be done for `decals_dr2` and `sdss_lost_set`, these missing values are cause this build to fail on `localhost`.

**Comment by murraycu** [comment]
Thanks, @CKrawczyk . Then I'll wait until the PR is complete, with a decision tree, or use of an older decision tree, and code to make it actually use the new survey.

**Comment by CKrawczyk** [comment]
@murraycu it will use the `ferengi` tree that was used before.

**Comment by CKrawczyk** [comment]
@astopy everything looks good to me!  Feel free to merge when you are ready.

**Comment by astopy** [comment]
@murraycu Do we need to coordinate deploying this with updates to the mobile apps?

**Comment by murraycu** [comment]
Do you really want to keep 3 surveys active? That's fairly unusual in the history of GalaxyZoo, I think. I suggest it could create some confusion for people to be faced with 3 different decision trees at random.

**Comment by murraycu** [comment]
> Do we need to coordinate deploying this with updates to the mobile apps?

As long as you don't turn off the existing surveys, you can go ahead without waiting for me. However, you might want to wait for me to run my tests on the new survey/decision-tree - my tests are rather more thorough.

**Comment by astopy** [comment]
@murraycu Thanks. The team want all three to be live. I'm going to go ahead and deploy this now rather than waiting, since they're eager for the new group to be live.

**Comment by murraycu** [comment]
This doesn't affect the mobile apps, but you might care that there are no icons specified for these new Ferengi2 and SDSS Lost Set surveys.

Maybe the SDSS Lost set icon should use the Sloan icon too:
https://github.com/zooniverse/Galaxy-Zoo/blob/master/css/common.styl#L24

The Decals DR2 survey (also live now) uses the decals icon, though I don't know where that is specified:
https://github.com/zooniverse/Galaxy-Zoo/blob/master/css/common.styl#L44

Maybe the Ferengi survey should have its own icon.

The actual icon files seem to be listed here:
https://github.com/zooniverse/Galaxy-Zoo/blob/master/css/profile.styl#L137
and the actual files are here:
https://github.com/zooniverse/Galaxy-Zoo/tree/master/public/images

**Comment by murraycu** [comment]
Otherwise, all my tests pass for these new surveys.

**Comment by murraycu** [comment]
The Examine page doesn't work for this new Ferengi 2 survey: https://github.com/zooniverse/Galaxy-Zoo/issues/272
(and maybe not for the SDSS Lost Set survey either.)

**Pull request merged**